extensions/interactions/NumericInput/validator.js

```diff
@@ -92,10 +88,12 @@ oppia.factory('NumericInputValidationService', [
               default:
             }
             for (var k = 0; k < ranges.length; k++) {
-              if (overlaps(ranges[k], range)) {
+              if (isEnclosedBy(range, ranges[k])) {
                 warningsList.push({
                   type: WARNING_TYPES.CRITICAL,
-                  message: 'Please ensure all numeric ranges are distinct.'
+                  message: (
+                    'Some rules will never be matched because' +
```

Comment:
Be more specific: In answer group X, rule Y will never be matched because it is made redundant by preceding rules. (Otherwise the creator won't know where to start looking in order to fix the error.)